explorer/explorer.go

```diff
@@ -296,13 +296,26 @@ func New(dataSource explorerDataSource, userRealIP bool) *explorerUI {
 			return t
 		},
 		"fancyDCR": func(v float64) []string {
```

Comment:
I like the nomenclature change to `clipped` here.

How about we rename this to `float64asFloatParts` and make its return value a string slice with three values just like `amountAsFloatParts`?  That way they can share the same render partial.